<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input3DCoordinates</class>
 <widget class="QWidget" name="input3DCoordinates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>{name1}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="sbx1"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>{name2}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="sbx2"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>{name3}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="sbx3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
